core/src/main/clojure/xtdb/information_schema.clj

```diff
@@ -199,6 +216,28 @@
      :nspowner nspowner
      :nspacl nil}))
 
+(def pg-ams [{:oid 2
+              :amname "heap"
+              :amhandler "heap_tableam_handler"
+              :amtype "t"
+             }
+             ;; FIXME this should prolly be something else (arrow-HMT-handler?)
```

Comment:
Can we try to use correct spelling.

Reply:
Likely `HAMTrie`

Reply:
probably you mean `prolly`? it's kind of like i18n - so not really keen on changing that. sorry.

Reply:
> probably you mean prolly? it's kind of like i18n

no it isn't, please fix it.